### ISSUE TITLE
[indent] class fields break indentations

### ISSUE THREAD

**Issue by gnuletik** [typescript-eslint/typescript-eslint#3150]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

```js
// .eslintrc.js

rules {
  indent: [ 'error', 2 ]
}
```

```js
// SomeFile.ts

export class Test {
  // should throws
      val = ''

  // throws Expected indentation of 2 spaces but found 6  indent
  // should work
  method2 () {}

  // works OK
  // should throws
      method6 {}
}
```

**Expected Result**

Line with an identation != 2 should throws an error

**Actual Result**

Line with an identation != 2 should do not throws an error

**Additional Info**

When removing the class field (line with `val ` ''`), it works as expected.
See https://github.com/tc39/proposal-class-fields

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `4.16.1` |
| `@typescript-eslint/parser`        | `4.16.1` |
| `TypeScript`                       | `4.1.3` |
| `ESLint`                           | `7.21.0` |
| `node`                             | `15.9.0` |

**Comment by bradzacher** [comment]
Please use the search.
There is a pinned issue on this. #1824

**Comment by gnuletik** [comment]
Thanks for the fast answer !

I checked the indent meta issue but did not found issues about class fields.

**Comment by bradzacher** [comment]
The issue title is just "Problems with the indent rule"...
Which implies **all** issues, not just the examples in the body.

**Comment by gnuletik** [comment]
Okay, sorry for the noise!
It may be worth to add a disclaimer in the meta issue asking not to create issues about indentation.

Thanks for the hard work!
